### DISCUSSION TITLE
Can this be used with a pony duffusion models? If so, how?

### DISCUSSION THREAD

**Discussion by VladossX** [vladmandic/sdnext#3138] [Q&A] [answered]
At the moment, I'm getting a noise 
![Screenshot_1](https://github.com/vladmandic/automatic/assets/42897215/368a94eb-9e9a-42e9-bf1b-7feaff2ebc39)

**Comment by vladmandic** [comment, marked as answer]
set clip skip to 1. recommendation to set it to 2 is wrong, that has been discussed many times.
